src/SFA.DAS.EmployerApprenticeshipsService.Domain/Models/Payments/Payment.cs

```diff
@@ -26,6 +26,49 @@ public class Payment
         public int? FrameworkCode { get; set; }
         public int? ProgrammeType { get; set; }
         public int? PathwayCode { get; set; }
-        public ContractType ContractType { get; set; }
+        public string PathwayName { get; set; }
+
+        public override bool Equals(object obj)
+        {
+            var payment = obj as Payment;
+
+            if (payment == null) return false;
+
+            if (Id != null || payment.Id != null)
+            {
+                if (!Id?.Equals(payment.Id) ?? false)
+                {
+                    return false;
+                }
+            }
+
+            if (Ukprn != payment.Ukprn) return false;
+            if (Uln != payment.Uln) return false;
+            if (EmployerAccountId != payment.EmployerAccountId) return false;
+            if (ApprenticeshipId != payment.ApprenticeshipId) return false;
+            if (DeliveryPeriodMonth != payment.DeliveryPeriodMonth) return false;
+            if (DeliveryPeriodYear != payment.DeliveryPeriodYear) return false;
+            if (CollectionPeriodId != payment.CollectionPeriodId) return false;
+            if (CollectionPeriodMonth != payment.CollectionPeriodMonth) return false;
+            if (CollectionPeriodYear != payment.CollectionPeriodYear) return false;
+            if (EvidenceSubmittedOn != payment.EvidenceSubmittedOn) return false;
+            if (EmployerAccountVersion != payment.EmployerAccountVersion) return false;
+            if (ApprenticeshipVersion != payment.ApprenticeshipVersion) return false;
+            if (FundingSource != payment.FundingSource) return false;
+            if (TransactionType != payment.TransactionType) return false;
+            if (Amount != payment.Amount) return false;
+            if (StandardCode != payment.StandardCode) return false;
+            if (FrameworkCode != payment.FrameworkCode) return false;
+            if (ProgrammeType != payment.ProgrammeType) return false;
+            if (PathwayCode != payment.PathwayCode) return false;
+            if (PathwayName != payment.PathwayName) return false;
```

Comment:
The string comparisons here will be case sensitive - don't know if this is deliberate.

Reply:
Yep that should be fine as they should match completely.